src/frontend/src/components/appHeaderComponent/components/HeaderMenu/index.tsx

```diff
@@ -15,11 +15,11 @@ export const HeaderMenu = ({ children }) => (
 );
 
 export const HeaderMenuToggle = ({ children }) => (
-  <DropdownMenuTrigger className="group inline-flex w-full items-center justify-center gap-1 rounded-md px-2 py-2 pr-0">
-    <div className="flex items-center gap-1 rounded-full group-hover:bg-muted">
+  <DropdownMenuTrigger className="group inline-flex w-full items-center justify-center gap-1 rounded-md pr-0">
+    <div className="-ml-3 flex items-center gap-1 rounded-lg px-2 py-1.5 group-hover:bg-muted">
```

Comment:
I think here would be better if we removed the ml-3 from the AppHeaderComponent instead of adding -ml-3 here

Reply:
done, good call